package.json

```diff
@@ -60,25 +60,25 @@
     "video.js": "^5.18.0 || ^6 || ^7"
   },
   "devDependencies": {
+    "@gkatsev/rollup-plugin-node-builtins": "^2.1.3",
     "conventional-changelog-cli": "^2.0.1",
-    "conventional-changelog-videojs": "^3.0.0",
+    "conventional-changelog-videojs": "^3.0.1",
     "doctoc": "^1.3.1",
     "husky": "^1.0.0-rc.13",
```

Comment:
Maybe you should just use [@videojs/generator-helpers](https://github.com/videojs/generator-helpers) for some of these packages? You may have to slightly change the husky config and the lint-staged config in package.json, but everything else should be the same.

Reply:
maybe next time.